### PR TITLE
Corrects example

### PR DESCRIPTION
**Pull request by karissademi**
Adds `data-equalizer-watch="foo"` in the example markup, reflecting the markup in the working example beneath it.

### CODE REVIEW

File: docs/pages/equalizer.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ In the below example, the first set of Equalizer elements have the value `foo`,
 ```html
 <div class="row" data-equalizer="foo">
   <div class="medium-4 columns" data-equalizer-watch="foo">
-    <div class="callout" data-equalizer="bar">
+    <div class="callout" data-equalizer-watch="foo" data-equalizer="bar">
```
Comment (IamManchanda) [inline docs/pages/equalizer.md]: Can you explain this @karissademi ??

Comment (IamManchanda) [inline docs/pages/equalizer.md]: Oh now i understood ..... thanks !

**Review by IamManchanda — APPROVED** [review]
LGTM 
@kball for merge or review!

File: docs/pages/equalizer.md (hunk 1 above)
Comment (karissademi) [inline docs/pages/equalizer.md]: 👍

**Pull request merged**